### ISSUE TITLE
External links are announced as "Link opens in a new window" by screenreaders, and nothing else

### ISSUE THREAD

**Issue by toolness** [18F/omb-eregs#655]
I was just looking through our custom `<Link>` component and noticed that, if the link's `href` attribute starts with `http:` or `https:`, it's given a `_blank` target and its `aria-label` attribute is set to `"Link opens in a new window"`.

Unfortunately, this means that screen-readers will *ignore* the actual text content within the link, and announce it *solely* as "Link opens in a new window", thus obscuring the actual destination of the link.  This can be easily verified on OSX/VoiceOver by visiting the site's homepage, pressing Cmd+F5 to activate VoiceOver, and then pressing tab until focus is on the "www.whitehouse.gov/omb" link.

<i>**Aside:** I encourage folks to actually try doing this if they've never used VoiceOver--just as one would rarely write CSS and not bother verifying that the page actually looks the way they want it to, we should prefer to manually verify that our accessibility fixes actually do what we want them to!</i>

Anyhow, I think it might be easiest to just get rid of the `aria-label` attribute entirely; I think (but am not certain) some screen readers can actually infer from the `target="_blank"` that the link will open in a new window and announce it to the user, in which case we don't want the user to hear *twice* (once from the screen reader and once from our `aria-label` text) that the link will open in a new window.

However, if we really want to keep this feature, we should probably do something like iterate through all the `children` of the link and grab their text content.

**Comment by cmc333333** [comment]
I agree that removing the label is a step up from where we're at, but let's hold off on closing this issue until we do some code history spelunking. I know we've had an issue related to this before and suspect we can find one in the eregs repos; I suspect the `aria-label` had something to do with policy compliance for external links, but it could be a broken implementation.

**Comment by toolness** [comment]
Ok, for now I've removed the `aria-label` attribute entirely in #649, but I'm leaving this issue open, as requested, so we can address a better solution later.

**Comment by carodew** [comment]
This sounds familiar ... 

Didn't we have the same issue on ATF eRegs?
